src/tracing/trace_flamegraph.h

```diff
@@ -0,0 +1,39 @@
+/*
+   +----------------------------------------------------------------------+
+   | Xdebug                                                               |
+   +----------------------------------------------------------------------+
+   | Copyright (c) 2002-2021 Derick Rethans                               |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 1.01 of the Xdebug license,   |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available at through the world-wide-web at                           |
+   | https://xdebug.org/license.php                                       |
+   | If you did not receive a copy of the Xdebug license and are unable   |
+   | to obtain it through the world-wide-web, please send a note to       |
+   | derick@xdebug.org so we can mail you a copy immediately.             |
+   +----------------------------------------------------------------------+
+ */
+#ifndef XDEBUG_TRACE_FLAMEGRAPH_H
+#define XDEBUG_TRACE_FLAMEGRAPH_H
+
+#define XDEBUG_TRACE_FLAMEGRAPH_COST_FACTOR 100000;
+
+#include "tracing_private.h"
+
+typedef struct _flamegraph_function
+{
+	xdebug_str *prefix;
+	int value;
+} flamegraph_function;
```

Comment:
If they're not prefixed with `xdebug_`, they can't be in the `.h` file - they either need to be in the `tracing_private.h` file, or preferably in a new `trace_flamegraph_private.h` file in case it's only required to be defined for the `trace_flamegraph.c` file.